multimodal/src/autogluon/multimodal/configs/optimization/adamw.yaml

```diff
@@ -3,6 +3,7 @@ optimization:
   learning_rate: 1.0e-4
   weight_decay: 0.001
   lr_choice: "layerwise_decay"
+  freeze_backbone: False
```

Comment:
Consider removing this hyperparameter and add `frozen_parts` under the `mmdet_image` model config.